docs/conceptual/node/tx-pool.md

```diff
@@ -2,12 +2,80 @@
+
+Full nodes in the network all have their own local transaction pool and transaction queues. Therefore, full nodes may have a different view of the queues and transactions in the network.
+
+Dependent on the runtime, strict ordering may not be a requirement. This could enable full nodes to implement different strategies on how they prioritize propagating transactions and how they're included into blocks.
+
+## Transaction Dependencies
+
+Transaction dependencies are defined by the `requires` and `provides` parameters in the `ValidTransaction` [struct](https://substrate.dev/rustdocs/master/sp_runtime/transaction_validity/struct.ValidTransaction.html) to build a dependency graph of transactions. Together with `priority`, this allows the pool to traverse the graph and produce a valid linear ordering of transactions to be propagated and included in the next block.
+
+For `Frame` based runtimes, an account based system is used for transaction ordering. Every signed transaction needs to contain a nonce, which is incremented by 1 every time a new transaction is made. For example, the first transaction from a new account will have `nonce=0` and the second transaction will have `nonce=1`.
+
+At minimum, `Frame` transactions have a `provide` tag of `encode(sender ++ nonce)` and a `require` tag of `encode(sender ++ (nonce -1)) if nonce > 1`. Transactions do not `require` anything if `nonce=0`. As a result, all transactions coming from a single sender will form a sequence in which they should be included.
+
+Substrate supports multiple `provide` and `require` tags, so custom runtimes can create alternate dependency (ordering) schemes.
+
+## Transaction Priority
+
+Transaction `priority` in the `ValidTransaction` [struct](https://substrate.dev/rustdocs/master/sp_runtime/transaction_validity/struct.ValidTransaction.html), helps determine the ordering of transactions that have their dependencies satisfied. The higher the priority number, the more important the transaction is.
+
+`Priority` helps define the linear ordering of a graph in the case of one transaction unlocking multiple dependent transactions. For example, if we have two (or more) transactions that have their dependencies satisfied, then we use `priority` to choose the order for them.
+
+
+For `Frame` based runtimes, `priority` is defined as the `fee` that the transaction is going to pay. For example:
+
+  * If we receive 2 transactions from *different* senders (with `nonce=0`), we use `priority` to determine which transaction is more important and included in the block first.
+
+  * If we receive 2 transactions from the *same* sender with an identical `nonce`, only one transaction can be included on-chain. We use `priority` to choose the transaction with a higher `fee` to store in the transaction pool.
+
```

Comment:
Remove empty line between list items for common rendering.
```suggestion
```

Reply:
I think you may have meant line 44 to delete. I've deleted line 44